usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -357,3 +362,50 @@ def post(self, request, pk, format=None):
         result = {"results": {q['name']: q['obligations_incurred_by_program_object_class_cpe__sum'] for q in queryset}}
 
         return Response(result)
+
+
+class FederalAccountsViewSet(APIView):
+    @cache_response()
+    def post(self, request, format=None):
+        """Return all high-level Federal Account information"""
+
+        limit = request.data.get("limit", 10)
+        page = request.data.get("page", 1)
+        sorting = request.data.get("sort", {'field': 'budgetary_resources', 'direction': 'desc'})
+        sort_field = sorting.get('field', 'budgetary_resources')
+        sort_direction = sorting.get('direction', 'asc')
+        filters = request.data.get('filters', {})
+        fy = filters.get('fy') or SubmissionAttributes.last_certified_fy()
+
+        lower_limit = (page - 1) * limit
+        upper_limit = page * limit
+
+        agency_subquery = ToptierAgency.objects.filter(cgac_code=OuterRef('agency_identifier'))
+        queryset = FederalAccount.objects.filter(
+            treasuryappropriationaccount__account_balances__final_of_fy=True,
+            treasuryappropriationaccount__account_balances__submission__reporting_period_start__fy=fy
+            ).annotate(
+            account_id=F('id'),
+            account_number=Concat(F('agency_identifier'), Value('-'), F('main_account_code')),
+            account_name=F('account_title'),
+            budgetary_resources=Sum(
+                'treasuryappropriationaccount__account_balances__obligations_incurred_total_by_tas_cpe'),
+            managing_agency=Subquery(agency_subquery.values('name')[:1]),
+            managing_agency_acronym=Subquery(agency_subquery.values('abbreviation')[:1]),
+        )
+
+        if sort_direction == 'desc':
+            queryset = queryset.order_by(F(sort_field).desc(nulls_last=True))
+        else:
+            queryset = queryset.order_by(F(sort_field).asc())
+
+        result = {'count': queryset.count(), 'limit': limit, 'page': page, 'fy': fy}
```

Comment:
If the federal account gets large, we may want revisit this `queryset.count()` function. For now there isn't an issue using it.